scripts/em_plots.py

```diff
@@ -90,37 +158,36 @@ def main():
     # Ensure no elements are missed
     assert all(merger_g1_mask | merger_g2_mask | merger_gX_mask) == 1
 
-# ===============================
-### comparison histogram ###
-# ===============================
+    # ===============================
+    ### comparison of shock and jet luminosities histogram ###
+    # ===============================
     fig = plt.figure(figsize=plotting.set_size(figsize))
+
     shock_bins = np.logspace(np.log10(mergers[:, 17].min()), np.log10(mergers[:, 17].max()), 50)
     jet_bins = np.logspace(np.log10(mergers[:, 18].min()), np.log10(mergers[:, 18].max()), 50)
+
     plt.hist(mergers[:, 17], bins = shock_bins, label = 'Shock')
     plt.hist(mergers[:, 18], bins = jet_bins, label = 'Jet', alpha = 0.8)
-    plt.axvline(10**46, linewidth = 1, linestyle = 'dashed', color = 'red', label = r"Fiducial L$_{QSO}$")
+    plt.axvline(lum_agn, color = 'black', linewidth = 1, linestyle = 'dashdot', label = r'L$_{AGN}$ =' + f"{lum_agn:.2e}")
```

Comment:
Please edit keyword arguments, here and throughout the script, to remove spaces surrounding the `=` following the [PEP 8 style guide](https://peps.python.org/pep-0008/#other-recommendations). See the 5th bullet point in the section "Other Recommendations". 
```suggestion
    plt.axvline(lum_agn, color='black', linewidth=1, linestyle='dashdot', label=r'L$_{AGN}$ =' + f"{lum_agn:.2e}")
```